.github/workflows/_build_truss_server_base_images_if_needed_shared.yml

```diff
@@ -1,6 +1,16 @@
 name: Build and Push Truss Base Images (if needed)
 
 on:
+  workflow_dispatch:
```

Comment:
This change should allow us to run this action [manually](https://github.com/basetenlabs/truss/actions/workflows/_build_truss_server_base_images_if_needed_shared.yml) via the gha UI